src/test/java/ConsoleUITest.java

```diff
@@ -0,0 +1,62 @@
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class ConsoleUITest {
+
+    private SpyConsole spyConsole;
+    private ConsoleUI cui;
+
+    @Before
+    public void setUp() throws Exception {
+        spyConsole = new SpyConsole();
+        cui        = new ConsoleUI(spyConsole);
```

Comment:
would prefer the variable to be more descriptive. Interestingly, just losing the c (i.e., ui) would be more descriptive. Otherwise, consoleUI would be fine.